README.md

```diff
@@ -1,30 +1,86 @@
+    * **[Building an App image](#building-an-app-image)**
+    * **[Sharing the App on the Hub](#sharing-apps-on-the-hub)**
+    * **[Running the App](#running-the-app)**
+- **[Example](#example)**
+    * **[App definition](#app-definition)**
+    * **[Using parameters](#using-parameters)**
+    * **[Building an App image](#building-an-app-image)**
+    * **[Sharing and running the App](#sharing-and-running-the-app)**
+- **[CNAB](#cnab)**
+- **[Installation](#installation)**
+    * **[Linux or macOS](#linux-or-macos)**
+    * **[Windows](#windows)**
+- **[Next steps](#next-steps)**
 
-Compose files do a great job of describing a set of related services. Not only
-are Compose files easy to write, they are generally easy to read as well.
-However, a couple of problems often emerge:
+## What are the benefits offered by Docker App?
 
-1. You have several environments where you want to deploy the application, with small configuration differences
-1. You have lots of similar applications
+* Simple management of Docker Apps across different teams and between different environments (Development/QA/Staging/Production)
+* Easy sharing of multi-service applications to container registries (e.g., [Docker Hub](https://hub.docker.com/) or [Docker Trusted Registry](https://docs.docker.com/ee/dtr/))
+* Having a clear separation of parameters to be modified at runtime 
+* Support for multiple orchestrators (Swarm or Kubernetes)
+* Provides the very same UX flow as the one for Docker images
+* Implements the [CNAB](https://cnab.io) industry standard
+* Offers full support of Docker Contexts
 
-Fundamentally, Compose files are not easy to share between concerns. Docker
-Application Packages aim to solve these problems and make Compose more useful
-for development _and_ production.
+## How does Docker App work?
 
-## Looking at an example
+The Docker App workflow is quite similar to the Docker image workflow. From an App definition, you can build an App image, share it on Docker Hub, and run your App on an orchestrator.
 
-Let's take the following Compose file. It launches an HTTP server which prints
-the specified text when hit on the configured port.
+![Image showing Docker CLI command flow](./DockerAppCLIcommands.png)
+
+## Using Docker App
+
+Four primary steps comprise the Docker App process:
+1. Writing an App Definition
+1. Building an App Image
+1. Sharing the App on the Hub (optional)
+1. Running the App
+
+### Writing an App definition
+
+The first step in using Docker App is to write the App definition. This definition can be created (1) from an existing Compose file using the `docker app init` command (2) via  a template from the Application Designer, or (3) from scratch. 
+
+The App definition is a .dockerapp folder that contains three distinct pieces: metadata, a service list, and the parameters. 
+
+metadata | Includes the App name and version; provides better traceability back to code changes when debugging issues
```

Comment:
Did you intend drawing a table here? Because otherwise it looks weird 
<img width="880" alt="Screen Shot 2019-11-12 at 18 11 00" src="https://user-images.githubusercontent.com/31478878/68693641-ef2a5f00-0577-11ea-98ba-9c6ebc62f9e7.png">

Reply:
yes it is supposed to render as a table

Reply:
I see it is not correctly rendered. Will double check.